src/components/projects-summary/projects-summary.html

```diff
@@ -99,4 +99,25 @@ <h2 class="secondary" ng-if="$ctrl.projects && $ctrl.projects.length > 0">Gettin
       </ul>
     </div>
   </div>
+  <div ng-if="$ctrl.recentlyViewedItems.length">
+    <h2 class="secondary">Recently Viewed</h2>
+    <div class="services-view">
+      <div class="services-view-container">
+        <div class="services-items-inner">
+          <div class="services-items-inner-shadow-covers">
+            <a href="" class="services-item" ng-repeat="item in $ctrl.recentlyViewedItems track by (item.resource | uid)"
```

Comment:
For example, when I click Python here, Python and WildFly swap places. The musical chairs dance is strange.

<img width="353" alt="screen shot 2017-04-20 at 2 09 07 pm" src="https://cloud.githubusercontent.com/assets/1167259/25245590/f2282548-25d2-11e7-9c65-fbb66705f525.png">